wsmaster/che-core-api-devfile/README.md

```diff
@@ -83,6 +86,11 @@ components:
     id: org.eclipse.che.editor.theia:1.0.0
 ```
 
+If it is missing then default one will be provided along with its default plugins.
+Default plugins will be provided also in case when editor is defined the same as default one.
```

Comment:
The default plugins will be provided also for an explicitly defined editor with the same ID as the default one (even if in a different version).